src/sentry/testutils/cases.py

```diff
@@ -967,8 +967,11 @@ def store_session(self, session):
             self._push_metric(session, "counter", "session", {"session.status": status}, +1)
             self._push_metric(session, "set", "user", {"session.status": status}, user)
 
-        if status != "ok":  # terminal
-            self._push_metric(session, "distribution", "session.duration", {}, session["duration"])
+        if status == "exited":  # Only collect healthy sessions
+            if session["duration"] is not None:
+                self._push_metric(
+                    session, "distribution", "session.duration", {}, session["duration"]
+                )
```

Comment:
This now must be reverted back to how it was before as we did not merge https://github.com/getsentry/relay/pull/1089 in its original form and decided to keep some "dead" metrics instead

Reply:
Changed the condition and added the `session.status` tag:

https://github.com/getsentry/sentry/blob/cd8e4a2184d65b810c83b090ca7e6d5c297aa063/src/sentry/testutils/cases.py#L970-L977